CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Draft
 - Hide Info in footer if no address is provided in Store Profile. Hide Brands in footer if Merchant has no brands [#1053](https://github.com/bigcommerce/cornerstone/pull/1053)
+- When a store has empty product inventory, we show a set of sample products on homepage. For sample products they has no URL and not clickable. They also don't have quick view. Check if it is sample product by id > 0.
```

Comment:
Can we remove this part `For sample products they has no URL and not clickable. They also don't have quick view. Check if it is sample product by id > 0.`. We should keep it simple & concise.